server/data-migrations/1707466537651-MoveVisibilityDisabledStatesToProperties.ts

```diff
@@ -4,7 +4,7 @@ import { EntityManager, MigrationInterface, QueryRunner } from 'typeorm';
 
 export class MoveVisibilityDisabledStatesToProperties1707466537651 implements MigrationInterface {
   public async up(queryRunner: QueryRunner): Promise<void> {
-    const componentTypes = ['TextInput', 'NumberInput', 'PasswordInput', 'Text'];
+    const componentTypes = ['TextInput', 'NumberInput', 'PasswordInput', 'Text', 'Button'];
```

Comment:
Will this work for when someon upgrades a version.? Dont we need a new migration here?

Reply:
Handled in https://github.com/ToolJet/ToolJet/pull/9665/files